src/components/ContactLinks/ContactLinks.scss

```diff
@@ -0,0 +1,38 @@
+@import "styles/colors.scss";
+
+.contact-links {
+  .title {
+    font-size: 2.5em;
+    font-weight: 350;
+  }
+
+  .text {
+    font-size: 1.5em;
+    font-weight: 300;
+    word-break: break-word;
+  }
+
+  .textIcon {
+    font-size: 1.5em;
+    margin-right: 0.2em;
+    font-weight: 300;
+    padding: 0;
+  }
+
+  .linebreak {
+    height: 3px;
```

Comment:
Should be down with tailwind

Reply:
Tailwind doesn't go small enough, `h-1`: 0.25rem looks odd imo.

Reply:
Hmm fair enough then.